backend/src/routes.js

```diff
@@ -1,32 +1,52 @@
 const express = require('express');
-const {User} = require("./Schema");
+const { User, NGO } = require("./Schema");
 const router = express.Router();
 
+//Route to get all users
+router.get('/users', async (req, res) => {
+    const users = await User.find({});
 
-router.get('/users', (req, res) => {
-    res.send(`<h1>No users yet :(</h1>`)
+    try {
+        res.status(200).json(users)
+    } catch (err) {
+        res.status(500).send(err)
+    }
 });
 
 
 //route to get a single user
-router.get('/users/:id', async (req,res) => {
-    try{
+router.get('/users/:id', async (req, res) => {
+    try {
         let user = await User.findById(req.params.id);
-        if (user){
+        if (user) {
             delete user.password;
             res.status(200).json({ user: user });
-        }else{
+        } else {
             res.status(404).send({ error: `User with id ${req.params.id} not found.` });
         };
-    }catch(err){
+    } catch (err) {
         res.status(500).send({ error: err });
-    };    
+    };
 });
 
 
 router.post('/users', (req, res) => {
     res.status(200).send(req.body)
 });
 
+router.post('/ngos', (req, res) => {
+
+    //req.body destructuring
+    const { name, image, webpage, description, main_representative, affinities, contact: { address, phone, contact_hours }
```

Comment:
I think this destructuring is kind of unnecessary. Please let me know if you had a reason for doing it.

Reply:
we made this to avoid writing many times "req.body"..